moveit_ros/moveit_node_interface/CMakeLists.txt

```diff
@@ -0,0 +1,39 @@
+cmake_minimum_required(VERSION 3.10.2)
+project(moveit_node_interface)
+
+find_package(ament_cmake REQUIRED)
+find_package(rclcpp REQUIRED)
+find_package(rclcpp_lifecycle REQUIRED)
+
+set(MOVEIT_LIB_NAME moveit_node_interface)
```

Comment:
Nit: I would remove this line and use `${PROJECT_NAME} `instead of `${MOVEIT_LIB_NAME}` since it is the same

Reply:
I made the change